docs/DeveloperGuide.md

```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
       than the number of assets/liabilities/policies listed).
       Expected: Similar to previous.
 
+### 7.6 Clearing client list
+
+1. Clearing all Clients in the home window.
+    1. Prerequisites: currently in the home window and there is at least 1 client.
```

Comment:
Client is currrently in the home window pls. or at least make ur c C

Reply:
done